ui/src/app/views/recent-work/component.html

```diff
@@ -0,0 +1,20 @@
+<div *ngIf="elementVisible()" class="recent-work" style="display: inline-block">
+  <div *ngIf="leftScrollVisible()" class="scroll-indicator" id="left-scroll"
+       (click)="moveDownList()">
```

Comment:
Nit: The naming here threw me. the div is all about left scrolling, but the click handler is not named similarly. `scrollLeft()` seems like a better handler name to me.

Same feedback for the name of `moveUpList()`